### PR TITLE
Cust 1386 styling search component

### PR DESCRIPTION
**Pull request by darushka**

### CODE REVIEW

File: client/app/common/directives/search/search.scss [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    & + .md-icon-button {
+      height: 20px;
+    }
+  }
+
+  .md-sidenav-right {
+    background-color: $body-background;
+  }
+
+  md-input-container {
```
Comment (againsley) [inline client/app/common/directives/search/search.scss]: any idea why github is highlighting this red? is it supposed to be .md-input-container?

Comment (jsmylnycky) [inline client/app/common/directives/search/search.scss]: @againsley   It is not a class, it is an element.  I think github likes to highlight css element selectors for some reason.  This isn't the first time I've seen this :)

**Comment by jsmylnycky** [comment]
@darushka  Please merge squash

**Comment by darushka** [comment]
Merged squash https://github.com/peachworks/peach.css/commit/c62fb1f91da933dfb71832392be14e8cae49ec43

**Comment by jsmylnycky** [comment]
Release: 2400dae v0.1.7

**Pull request closed without merging**